packages/node-sdk/src/client.ts

```diff
@@ -77,6 +78,17 @@ type BulkEvent =
       context?: TrackingMeta;
     };
 
+/**
+ * A context with tracking option.
+ **/
+type ContextWithTracking = Context & {
+  /**
+   * Enable tracking for the context.
+   * If set to `false`, tracking will be disabled for the context. Default is `true`.
+   */
+  enableTracking?: boolean;
```

Comment:
Looks like we can remove the question mark here since we set the default everywhere.

Reply:
You're right, I had to use `interface` instead of `type`